libsplinter/src/rest_api/auth/rbac/rest_api/actix_web_1/assignments.rs

```diff
@@ -54,6 +54,20 @@ pub fn make_assignments_resource(
         })
 }
 
+pub fn make_assignment_resource(
+    role_based_auth_store: Box<dyn RoleBasedAuthorizationStore>,
+) -> Resource {
+    let get_store = role_based_auth_store;
+    Resource::build("/authorization/assignments/{identity_type}/{identity}")
```

Comment:
This should be in a separate file since it's a different endpoint.

Reply:
Will fix in a follow-up PR, as the same needs to be done for the roles/role routes